robocop/checkers/misc.py

```diff
@@ -89,8 +89,8 @@
         rule_id="0908",
         name="if-can-be-used",
         msg="'{{ run_keyword }}' can be replaced with IF block since Robot Framework 4.0",
-        severity=RuleSeverity.INFO,
-        version=">=4.0",
+        severity=RuleSeverity.WARNING,
```

Comment:
You can leave it with INFO (since we decided to deprecate this rule for versions above 4).